packages/exec/src/exec.ts

```diff
@@ -14,7 +16,7 @@ import * as tr from './toolrunner'
 export async function exec(
   commandLine: string,
   args?: string[],
-  options?: im.ExecOptions
+  options?: ExecOptions
```

Comment:
@ericsciple - I don't think this introduces a compat issue since it's a build time thing but I want to make sure.

Reply:
i think either should work since they have all the same members. i'll test shortly...

Reply:
Yes, either works. I added another source file sibling to exec:

```js
import * as im from './interfaces'
import * as exec from './exec'

let options: im.ExecOptions = {}
exec.exec("foo", [], options)

let options2: exec.ExecOptions = {}
exec.exec("foo", [], options2)
```

Both ways compile. I think it's because they have all the same members.

Also this is the pattern i used in the glob module. Callers reference interface re-exported from the main file. Internal code references the internal interface.